client/components/onboarding-select-control/index.js

```diff
@@ -0,0 +1,195 @@
+/** @format */
+
```

Comment:
Need to add a comment here linking to the original [CustomSelectControl](https://github.com/WordPress/gutenberg/tree/7aa042605ff42bb437e650c39132c0aa8eb4ef95/packages/components/src/custom-select-control) code.